app/javascript/packs/application.js

```diff
@@ -2,4 +2,29 @@ require("@rails/ujs").start()
 require("turbolinks").start()
 require("@rails/activestorage").start()
 require("channels")
-require("jquery")
\ No newline at end of file
+require("jquery")
+
+
+$( document ).ready(function() {
```

Comment:
I'm almost sure we are using this style: `$(document)`

Reply:
Here is the commit 3f1d995